client/src/redux/project/project.types.ts

```diff
@@ -1,15 +1,21 @@
 export interface Project {
-	_id: string;
+	_id?: string;
```

Comment:
Id is optional because when we send project details to the server we don't know what they are. At the same time, we know we won't receive a project if there's no id attached to it so this is pretty safe